src/WTG.Analyzers.Test/TestData/VarAnalyzer/Out/Result.cs

```diff
@@ -27,7 +27,7 @@ public void N()
 	void Ambiguous(out double value) => value = 0;
 	void SemiAmbiguous(out int value1, out int value2, out bool value3) => value3 = (value1 = value2 = 0) == 0;
 	void SemiAmbiguous(out double value1, out double value2, out bool value3) => value3 = (value1 = value2 = 0) == 0;
-	void Generic<T>(out T value) => value = default(T);
+	void Generic<T>(out T value) where T : struct => value = default(T);
```

Comment:
Why is this necessary? It because `out T` is not nullable?

Reply:
yep, but weirdly, it [only generated the error when running the test in net6](https://github.com/WiseTechGlobal/WTG.Analyzers/runs/5594525147?check_suite_focus=true).

Reply:
that's very weird